codecov.yml

```diff
@@ -13,9 +13,12 @@ coverage:
         # Coverage can drop by 0.5% before it is a failure.
         threshold: 0.5%
         paths:
-          - vowpalwabbit
-          - explore
-          - "!test"
+          - "vowpalwabbit/"
```

Comment:
what about explore?

Reply:
Explore is no longer a top level directory, its a lib under vowpalwabbit